themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -149,6 +149,20 @@
 
     </div>
   </div>
+  <script>
+    var forms = document.getElementsByTagName('form');
+
+    for (var i = 0; i < forms.length; i++) {
+        var form = forms[i];
+        form.addEventListener('submit', preventDoubleClick);
+    }
+
+    function preventDoubleClick(event) {
+        if (event.submitter) {
+            event.submitter.disabled = true;
+        }
+    }
+  </script>
```

Comment:
We can prevent the test suite from executing this code by adding a `type=module` to the script tag. Since HtmlUnit doesn't support this type it will not run the code, but modern browsers will. Another advantage of this aproach is that we automatically opt-in to strict mode, and also prevent leaking variables to the global scope (`window`).

I think we can also move to using [`document.forms`](https://developer.mozilla.org/en-US/docs/Web/API/Document/forms), which removes the need to query the DOM manually for form elements.

Given all this, we can also restore the more modern syntax that we had before, giving us something like this:

```suggestion
  <script type="module">
      for (const form of document.forms) {
        form.addEventListener('submit', preventDoubleClick);
      }

      function preventDoubleClick({ submitter }) {
          if (submitter) {
              submitter.disabled = true;
          }
      }
  </script>
```